src/components/modals/ModalButtons/ModalButtons.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import ModalButtons from './ModalButtons';
+
+export default {
+  title: 'Modals/ModalButtons',
+  component: ModalButtons,
+  argTypes: {},
+} as ComponentMeta<typeof ModalButtons>;
+
+const Template: ComponentStory<typeof ModalButtons> = (args) => (
+  <ModalButtons {...args} />
+);
+
+export const Normal = Template.bind({});
+Normal.args = {
+  withCancel: false,
+};
+
+export const WithCancel = Template.bind({});
+WithCancel.args = {
+  withCancel: true,
```

Comment:
`withCancel` is true by default so it is more like Normal case
Let's swap it with story above, so there are:
1) Normal - without arguments
2) WithoutCancel - withCancel=false